setup.py

```diff
@@ -14,13 +14,35 @@
 
 from setuptools import setup, find_packages
 from pip.req import parse_requirements
+import os
+
+def read(filename):
+    return open(os.path.join(os.path.dirname(__file__), filename)).read()
 
 requirements = [str(r.req) for r in parse_requirements('requirements.txt')]
 
 setup(name='haas',
       version='1.0',
+      description='HaaS is a low-level tool for reserving physical machines'
+                  ' and connecting them via isolated networks.',
+      long_description=read('README.rst'),
```

Comment:
Maybe we should have another document which would be more informative to the user or just remove the reference to readme.rst file.